projects/spectator/src/lib/spectator-routing/create-factory.ts

```diff
@@ -86,5 +88,18 @@ function createSpectatorRouting<C>(options: Required<SpectatorRoutingOptions<C>>
 
   const component = setProps(fixture.componentInstance, props);
 
-  return new SpectatorRouting(fixture, debugElement, component, TestBed.get(Router), TestBed.get(ActivatedRoute));
+  /**
+   * Back compatibility, angular under 8 version doesnt have a inject function
+   */
+  if (!TestBed.inject) {
```

Comment:
?

Reply:
Mistake in comment, `inject` in TestBed was added in Angular 9 version so we need this backward compatibility. I'm going to correct the comments

Reply:
updated comments